source/Random.h

```diff
@@ -32,6 +32,15 @@ class Random {
 	
 	static double Real();
 	
+	// Functions for getting random numbers that are "just for effect", meaning
+	// that calling them doesn't change the RNG used for more important game
+	// state. Examples are random pitch changes in sound effects and the
+	// generation of the background star field at program startup.
+	static uint32_t EffectInt();
+	static uint32_t EffectInt(uint32_t modulus);
+
+	static double EffectReal();
+	
```

Comment:
I wish I could think of a better name for these. Ideally the RNG wouldn't be a static class and then it'd be easier to control changes to the game state, but that would be a more extensive change.